src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -514,6 +514,20 @@ class GroupReadsByUmi
     }
   }
 
+  /** The error message when reads are out of order. */
+  private def getReadsOutOfOrderErrorMessage(r1: SamRecord, r2: SamRecord): String = {
+    val errorMessage = if (this.rawTag == "MI") {
+      s"Using 'MI' tag as the 'raw tag', are you sure this is correct?  Did you mean the 'RX' tag?"
```

Comment:
Should we just check this at startup and emit a warning?

Reply:
@tfenne What UmiAwareMarkDuplicatesWithMateCigar is doing does appear to be very wrong based on what you've shown.  Could you share that bam, I would like to try to fix this issue in Picard.

Reply:
@fleharty I don't think I can share that BAM, but I can probably use what I figured out to generate a small test case that I can pass along.  I'll try and do that in the next couple of days.

Reply:
@tfenne I just wanted to follow up on this.  Could you generate the small test case?
Thanks!